components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -184,6 +189,14 @@ func (r *OpenshiftNotebookReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	// Call the Rolebinding reconciler
+	if strings.ToLower(strings.TrimSpace(os.Getenv("SET_PIPELINE_RBAC"))) == "true" {
```

Comment:
We should have a helper function for this, to be used everywhere we're checking a bool env var flag. I'll open a ticket from this comment to remember.

One possibility would be to use https://pkg.go.dev/strconv#ParseBool there; and another possibility would be to accept "true" as true and any other string as "false"; not sure if we should have defaults for when env variable is not present (current code takes that as false, if I am looking correctly).

Reply:
* https://github.com/opendatahub-io/kubeflow/issues/440